src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import LoginForm from "./components/LoginForm";
+import { Switch, Route } from "react-router-dom";
+import NewQuest from "./components/NewQuest";
+import "./App.css";
+
+const App = () => {
+  return (
+    <>
+      <h1>
+        <span id="re">re</span>
+        Quest
+      </h1>
+      <Switch>
+        <Route exact path="/" component={LoginForm} />
+        <Route exact path="/newquests" component={NewQuest} />
```

Comment:
the naming here should probably request, since it's from the requester's perspective